recipes/arm-gnu-toolchain/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.15)
+
+# Required because Conan does not emit `set(CMAKE_SYSTEM_NAME "GENERIC")` or
+# `set(CMAKE_SYSTEM_PROCESSOR "ARM")` causing host flags to be leacked to CMake
+# in the conan_toolchain.cmake
+set(CMAKE_TOOLCHAIN_FILE ../toolchain.cmake)
```

Comment:
Why is the whole toolchain needed? This does not test the package on a standalone way because it includes the file from the CCI tree itself.

Also, is there no way to define the missing variables using the CMake helpers in Conan? I would have expected it to be doable